cumulusci/tasks/command.py

```diff
@@ -68,28 +81,46 @@ def _handle_returncode(self, returncode, stderr):
             self.logger.error(message)
             raise CommandException(message)
 
-    def _run_command(self, env):
+    def _run_command(self, env, command=None, output_handler=None, return_code_handler=None):
```

Comment:
Why the optional args? The only one I see used to override is `command` and in that case it's confusing because you also have a `command` option available to the user, but it is hard-coded to override.